### PR TITLE
ISO-DE Keymap for Dactyl Manuform 6x7

### PR DESCRIPTION
**Pull request by andi4000**
## Description
Add ISO-DE keymap for Dactyl Manuform 6x7

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by waffle87 — CHANGES_REQUESTED** [review]
`keyboards/handwired/dactyl_manuform/6x7/keymaps/iso-de/README.md` should be renamed to lowercase.

**Comment by andi4000** [comment]
> `keyboards/handwired/dactyl_manuform/6x7/keymaps/iso-de/README.md` should be renamed to lowercase.

Thanks for reviewing! The file is renamed to lowercase.

File: keyboards/handwired/dactyl_manuform/6x7/keymaps/iso-de/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## Layout
+
+![Dactyl Manuform 6x7 ISO-DE](https://i.imgur.com/LUlCGVd.png)
+
+## Flashing
+```bash
+qmk flash -kb handwired/dactyl_manuform/6x7 -km iso-de
+```
+Reset the keyboard when prompted to enter bootloader.
```
Comment (waffle87) [inline keyboards/handwired/dactyl_manuform/6x7/keymaps/iso-de/readme.md:15]:
```suggestion
Hardware build log: [andi4000/dactyl-keyboard](https://github.com/andi4000/dactyl-keyboard)

Please note the wiring differences with the default keymap in `config.h`.

## Layout

![Dactyl Manuform 6x7 ISO-DE](https://i.imgur.com/LUlCGVdh.png)

## Flashing
```bash
qmk flash -kb handwired/dactyl_manuform/6x7 -km iso-de
```

Reset the keyboard when prompted to enter bootloader.
```

Comment (andi4000) [reply keyboards/handwired/dactyl_manuform/6x7/keymaps/iso-de/readme.md]: Thanks for the suggestion! I have updated the formatting on the readme.

**Review by tzarc — APPROVED** [review]

**Pull request merged**